### PR TITLE
EZP-25775: Checkbox content edition support

### PR DESCRIPTION
**Pull request by glye**
> Implements https://jira.ez.no/browse/EZP-25775
> Status: Ready for review

Pretty much the simplest possible mapper.

Moved to feature branch from master: https://github.com/ezsystems/repository-forms/pull/81

### CODE REVIEW

File: lib/FieldType/Mapper/CheckboxFormMapper.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+     * @var \eZ\Publish\API\Repository\FieldTypeService
+     */
+    private $fieldTypeService;
+
+    public function __construct(FieldTypeService $fieldTypeService)
+    {
+        $this->fieldTypeService = $fieldTypeService;
+    }
+
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
```
Comment (jjCavalleri) [inline lib/FieldType/Mapper/CheckboxFormMapper.php]: The signature of mapFieldValueForm method does not match the used signature in other mappers (for instance: [IntegerFormMapper](https://github.com/ezsystems/repository-forms/blob/master/lib/FieldType/Mapper/IntegerFormMapper.php#L19), [TextLineFormMapper](https://github.com/ezsystems/repository-forms/blob/master/lib/FieldType/Mapper/TextLineFormMapper.php#L34))
Would you agree on changing the variable name from **$fieldForm** to **$fieldDefinitionForm**?

**Comment by bdunogier** [comment]
Detail: no need for "fix", it is not an issue :-)

That aside, I have tested the branch, and I'm getting an array to string conversion error if the selection isn't a multiple one. It works if it

**Comment by bdunogier** [comment]
Note: we can squash this when merging, imho.

**Comment by yannickroger** [comment]
+1 once passes also

**Pull request merged**